nonio-ios/Sources/APIs/Models/Post.swift

```diff
@@ -12,8 +12,8 @@ struct Post: Codable {
     let score: Int
     let commentCount: Int
     let tags: [PostTag]
-    let width: Int?
-    let height: Int?
+    let width: CGFloat?
```

Comment:
Why the shift to floats here? Not necessarily opposed/asking for a change, just curious.

Reply:
Because it's more convenient to work with UI layer eg, `CGSize`